testing/scripts/jupyterlab-testing-scripts.md

```diff
@@ -0,0 +1,98 @@
+# JupyterLab Testing Scripts
+
+This is a series of [testing scripts](). designed to mimic common manual accessibility tests in an automated testing setting for JupyterLab. They currently reference [WCAG 2.1](), but ideally will be updated for future versions of WCAG or to align with [ACT-rules]().
```

Comment:
There are multiple missing links

Reply:
Thank you for catching that! I'm really embarrassed by this one. 🙃

Reply:
it happens 🤗